src/stan/mcmc/hmc/xhmc/adapt_dense_e_xhmc.hpp

```diff
@@ -0,0 +1,55 @@
+#ifndef STAN_MCMC_HMC_xhmc_ADAPT_DENSE_E_XHMC_HPP
+#define STAN_MCMC_HMC_xhmc_ADAPT_DENSE_E_XHMC_HPP
+
+#include <stan/interface_callbacks/writer/base_writer.hpp>
+#include <stan/mcmc/stepsize_covar_adapter.hpp>
+#include <stan/mcmc/hmc/xhmc/dense_e_xhmc.hpp>
+
+namespace stan {
+  namespace mcmc {
+    /**
+      * The No-U-Turn sampler (XHMC) with multinomial sampling
```

Comment:
The following * go under the first, not the second one.  Emacs just does this automatically.